### PR TITLE
Moving Average Aggregate update with MAD intervals instead of 4

### PR DESCRIPTION
**Pull request by FelixGaudin**

### CODE REVIEW

File: draft-ietf-ippm-responsiveness.md [hunk 1]
```diff
@@ -535,7 +535,7 @@ Where
 - i: The index of the current interval. The variable i is initialized to 0 when the algorithm begins and
   increases by one for each interval.
 - moving average aggregate goodput at interval p: The number of total bytes of data transferred within
-  interval p and the three immediately preceding intervals, divided by four times the interval duration.
+  interval p and the MAD - 1 immediately preceding intervals, divided by MAD times the interval duration.
```
Comment (hawkinsw) [inline draft-ietf-ippm-responsiveness.md:538]: Great catch! I do not have the power to accept this PR but wanted to say +1.

File: draft-ietf-ippm-responsiveness.txt [hunk 2]
```diff
@@ -211,17 +211,17 @@ Internet-Draft   Responsiveness under Working Conditions      March 2023
    way as in-network bufferbloat.
 
    Finally, it is crucial to recognize that significant queueing only
-   happens on entry to the lowest-capacity (or "bottleneck") hop on a
+   happens on entry to the lowest-capacity (or “bottleneck”) hop on a
```
Comment (hawkinsw) [inline draft-ietf-ippm-responsiveness.txt:214]: Only slightly worried about this change (and the similar ones that follow). It looks like the rendering software that you are using has attempted to replace simple "s and 's with smart quotes. I do not 100% know how the IETF prefers to handle that in their formatting. Perhaps @cpaasch knows.

Comment (cpaasch) [reply draft-ietf-ippm-responsiveness.txt:214]: This will go away when I recompile

**Comment by cpaasch** [comment]
Thanks, Felix.

**Pull request merged**